tobe_blog_web/src/components/common/InfiniteScrollList.tsx

```diff
@@ -17,7 +17,7 @@ export default function InfiniteScrollList<T>(
   useEffect(() => {
     const scrollEvent = () => {
       if (!hasMore || loading) return;
-      if (document.documentElement.scrollHeight <= document.documentElement.clientHeight + document.documentElement.scrollTop) {
+      if (document.documentElement.scrollHeight * 0.8 <= document.documentElement.clientHeight + document.documentElement.scrollTop) {
```

Comment:
Add 0.8 to make the loading more smooth, not wait to load next page untill scroll to the page bottom